### PR TITLE
feat: Stargaze spec from Impulse

### PR DESCRIPTION
**Pull request by MichaelLLC**
Hello, I'm Michael, founder of Impulse.expert, specializing in high-performance, secure, and scalable node operations across 21 chains, maintaining 99% uptime. I've been actively involved with Lava since October 21, 2022, contributing to docs, socials and automation scripts. I'm deeply engaged in community support and project development. The custom automation and monitoring systems ensure reliable operations. I'm eager to maintain the Stargaze RPC spec on Lava and continue supporting the community with my expertise.

Commission: 2.5%:

    1% - Support and run a provider
    0.5% - Inherit, maintain and update Stargaze spec
    1% - Provider support

Discord: michaels1473

### CODE REVIEW

File: mainnet-1/specs/stargaze.json [hunk 1]
```diff
@@ -0,0 +1,787 @@
...
+        "reliability_threshold": 268435455,
+        "data_reliability_enabled": true,
+        "block_distance_for_finalized_data": 0,
+        "blocks_in_finalization_proof": 1,
+        "average_block_time": 5800,
+        "allowed_block_lag_for_qos_sync": 2,
+        "shares": 1,
+        "min_stake_provider": {
+          "denom": "ulava",
+          "amount": "15000000000"
```
Comment (shleikes) [inline mainnet-1/specs/stargaze.json:23]: This should be 5K lava (= 5000000000 ulava)

File: mainnet-1/specs/stargaze.json [hunk 2]
```diff
@@ -0,0 +1,787 @@
...
+                }
+              }
+            ]
+          }
+        ],
+        "contributor": [
+          "lava@1jnsszemt05kcxd4pf3uzewl9xqtzfpfg65960w"
+        ],
+        "contributor_percentage": "0.025",
+        "identity": "7C0AE2E5F08269E6"
```
Comment (shleikes) [inline mainnet-1/specs/stargaze.json:687]: This should be the logo of Axelar, if you don't have it, you can leave empty (= removed)

File: mainnet-1/specs/stargaze.json [hunk 3]
```diff
@@ -0,0 +1,787 @@
...
+                ]
+              }
+            ]
+          }
+        ],
+        "contributor": [
+          "lava@1jnsszemt05kcxd4pf3uzewl9xqtzfpfg65960w"
+        ],
+        "contributor_percentage": "0.025",
+        "identity": "7C0AE2E5F08269E6"
```
Comment (shleikes) [inline mainnet-1/specs/stargaze.json:782]: This should be the logo of Axelar, if you don't have it, you can leave empty (= removed)

File: mainnet-1/specs/stargaze.json [hunk 4]
```diff
@@ -0,0 +1,787 @@
...
+        "reliability_threshold": 268435455,
+        "data_reliability_enabled": true,
+        "block_distance_for_finalized_data": 0,
+        "blocks_in_finalization_proof": 1,
+        "average_block_time": 5800,
+        "allowed_block_lag_for_qos_sync": 2,
+        "shares": 1,
+        "min_stake_provider": {
+          "denom": "ulava",
+          "amount": "15000000000"
```
Comment (shleikes) [inline mainnet-1/specs/stargaze.json:705]: This should be 5K lava (= 5000000000 ulava)

**Review by shleikes — APPROVED** [review]

File: mainnet-1/specs/stargaze.json (hunk 1 above)
Comment (MichaelLLC) [reply mainnet-1/specs/stargaze.json]: The field value has been updated [f945313](https://github.com/lavanet/lava-config/pull/24/commits/f945313bd76273364a1dd62f8aa8357432ed92d8)

File: mainnet-1/specs/stargaze.json (hunk 2 above)
Comment (MichaelLLC) [reply mainnet-1/specs/stargaze.json]: The field value has been updated [f945313](https://github.com/lavanet/lava-config/pull/24/commits/f945313bd76273364a1dd62f8aa8357432ed92d8)

File: mainnet-1/specs/stargaze.json (hunk 3 above)
Comment (MichaelLLC) [reply mainnet-1/specs/stargaze.json]: The field value has been updated [f945313](https://github.com/lavanet/lava-config/pull/24/commits/f945313bd76273364a1dd62f8aa8357432ed92d8)

File: mainnet-1/specs/stargaze.json (hunk 4 above)
Comment (MichaelLLC) [reply mainnet-1/specs/stargaze.json]: The field value has been updated [f945313](https://github.com/lavanet/lava-config/pull/24/commits/f945313bd76273364a1dd62f8aa8357432ed92d8)

**Review by ignacio-rsg — APPROVED** [review]

**Pull request merged**